static/frame.js

```diff
@@ -1,13 +1,21 @@
-video_name = "test_vid"
-total_frames = 99
+video_name = "test_vid";
+total_frames = 99;
 
+var images = [];
+function frame_path(frame) {
+    block_0 = Math.floor(frame / 10000);
+    block_1 = Math.floor(frame / 100);
+    return `/static/videos/${video_name}/${block_0}/${block_1}/${frame}.jpg`;
+}
 function frame_url(frame) {
-    return `url(` + frame_path(frame)`)`;
+    return `url(` + frame_path(frame) + `)`;
```

Comment:
The backticks are actually ES6 string interpolation, so you should use that instead of adding strings together. Something like `` `url(${frame_path(frame)})` ``